src/KeyboardLayout_hu_HU.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+ * Standard HU keyboard layout.
+ */
+
+#include "KeyboardLayout.h"
+
+extern const uint8_t KeyboardLayout_hu_HU[128] PROGMEM =
+{
+	0x00,          // NUL
+	0x00,          // SOH
+	0x00,          // STX
+	0x00,          // ETX
+	0x00,          // EOT
+	0x00,          // ENQ
+	0x00,          // ACK
+	0x00,          // BEL
+	0x2a,          // BS  Backspace
+	0x2b,          // TAB Tab
+	0x28,          // LF  Enter
+	0x00,          // VT
+	0x00,          // FF
+	0x00,          // CR
+	0x00,          // SO
+	0x00,          // SI
+	0x00,          // DEL
+	0x00,          // DC1
+	0x00,          // DC2
+	0x00,          // DC3
+	0x00,          // DC4
+	0x00,          // NAK
+	0x00,          // SYN
+	0x00,          // ETB
+	0x00,          // CAN
+	0x00,          // EM
+	0x00,          // SUB
+	0x00,          // ESC
+	0x00,          // FS
+	0x00,          // GS
+	0x00,          // RS
+	0x00,          // US
+
+	0x2c,          // ' '
+	0x21|SHIFT,    // !
+	0x1f|SHIFT,    // "
+	0x1b|ALT_GR,   // #
+	0x33|ALT_GR,   // $
+	0x22|SHIFT,    // %
+	0x06|ALT_GR,   // &
+	0x1e|SHIFT,    // '
+	0x25|SHIFT,    // (
+	0x26|SHIFT,    // )
+	0x38|ALT_GR,   // *
+	0x20|SHIFT,    // +
+	0x36,          // ,
+	0x38,          // -
+	0x37,          // .
+	0x23|SHIFT,    // /
+
+	0x35,          // 0
+	0x1e,          // 1
+	0x1f,          // 2
+	0x20,          // 3
+	0x21,          // 4
+	0x22,          // 5
+	0x23,          // 6
+	0x24,          // 7
+	0x25,          // 8
+	0x26,          // 9
+
+	0x37|SHIFT,    // :
+	0x36|ALT_GR,   // ;
+	0x32|ALT_GR,   // <
+	0x24|SHIFT,    // =
+	0x1d|ALT_GR,   // >
+	0x36|SHIFT,    // ?
+	0x19|ALT_GR,   // @
+
+	0x04|SHIFT,    // A
+	0x05|SHIFT,    // B
+	0x06|SHIFT,    // C
+	0x07|SHIFT,    // D
+	0x08|SHIFT,    // E
+	0x09|SHIFT,    // F
+	0x0a|SHIFT,    // G
+	0x0b|SHIFT,    // H
+	0x0c|SHIFT,    // I
+	0x0d|SHIFT,    // J
+	0x0e|SHIFT,    // K
+	0x0f|SHIFT,    // L
+	0x10|SHIFT,    // M
+	0x11|SHIFT,    // N
+	0x12|SHIFT,    // O
+	0x13|SHIFT,    // P
+	0x14|SHIFT,    // Q
+	0x15|SHIFT,    // R
+	0x16|SHIFT,    // S
+	0x17|SHIFT,    // T
+	0x18|SHIFT,    // U
+	0x19|SHIFT,    // V
+	0x1a|SHIFT,    // W
+	0x1b|SHIFT,    // X
+	0x1d|SHIFT,    // Y
+	0x1c|SHIFT,    // Z
+
+	0x09|ALT_GR,   // [
+	0x14|ALT_GR,   // bslash
+	0x0a|ALT_GR,   // ]
+	0x20|ALT_GR,   // ^
```

Comment:
I believe this one is wrong. According to [kbdlayout.info][], this key combination yields a [dead][] circumflex accent (can you confirm this?), whereas this position in the table should map to the ASCII caret (code point U+005E), which is not a dead key, but a character in itself.

If this is indeed the case, this position should be marked as `0x00`. C.f. this explanation on pull request #53:

> Characters requiring hitting multiple keys _consecutively_ (like dead circumflex + space → `^` on some layouts) are not supported for the same reason. [namely: this would require growing the ASCII map to 16&nbsp;bits per cell, which would cost 128&nbsp;extra bytes of flash.]

[kbdlayout.info]: http://kbdlayout.info/KBDHU/
[dead]: https://en.wikipedia.org/wiki/Dead_key

Reply:
Oh my, I barely understand 10% of what you said, but I think you mean that '^' requires me to press AltGr + 3 and than another button to be typed. Like a space (or any other).

Other chars you mentioned do not require this extra button press.

Should I remove the '^' part? Also I checked and unfortanetely I did not test for that char. The test code:
````
const char* specialChars[] = {
    "_ underscore",
    "? question mark",
    "! exclamation mark",
    "# hashtag",
    "% percent",
    "* asterisk",
    "$ dollar sign",
    "@ at symbol",
    "& ampersand",
    "| vertical bar",
    "\\ backslash",
    "/ forward slash",
    "\" double quote",
    "' single quote",
    "` backtick",
    "~ tilde"
    "; semicolon",
    ": colon",
    ", comma",
    ". dot",
    "- hyphen",
    "+ plus",
    "= equals",
    "() parentheses",
    "[] brackets",
    "{} braces",
    "<> angle brackets",
    "§ section sign",
  };
````ˇ

I made it in a hurry... It's a bit sloppy :/

Reply:
> Should I remove the '^' part?

I think you can leave it after all, as on some variations of the Hungarian layout AltGr+3 gives `^` without requiring an extra space.